src/AppStore.ts

```diff
@@ -50,14 +33,23 @@ export class AppStore {
         }
     }
 
-    @computed get undismissedSiteErrors() {
+    @computed get undismissedSiteErrors(): SiteError[] {
         return _.filter(this.siteErrors.slice(), err => !err.dismissed);
     }
 
     @computed get isErrorCondition() {
```

Comment:
as per your edit, maybe lets call this `isFatalError` or such

Reply:
It's not necessarily a fatal error.  It could be a dialog error (any i may introduce an alert error also) which is dimissiable and allows user to continue to use site